game-core/src/main/java/games/strategy/triplea/ui/screen/UnitsDrawer.java

```diff
@@ -110,14 +111,22 @@ public void draw(
       throw new IllegalStateException("Type not found:" + unitType);
     }
     final GamePlayer owner = data.getPlayerList().getPlayerId(playerName);
+    final boolean damagedImage = damaged > 0 || bombingUnitDamage > 0;
+    final boolean disabledImage = disabled;
```

Comment:
nit, what's the motivation to re-assign `disabled` to a new variable?

Reply:
Attempt to ensure that the same values passed in the getImage call would be used in the log message.

Reply:
Is it not thread safe?
If we're trying to protect against concurrency, what's to keep the value from changing before this line? (in which case, unless I'm mistaken, the log message is also incorrect)

Reply:
It should be thread safe.

This was to protect a developer from changing the condition for the call and not changing the condition for the log call.  But I've removed the disabledImage variable and now just use disabled.